README.md

```diff
@@ -97,13 +97,31 @@ You can find those example applications under `examples/`.
 
 ### Run the Trace Server
 
-In order to open traces, you need a trace server running on the same machine as the trace extension. You can download the [Eclipse Trace Compass server](https://download.eclipse.org/tracecompass.incubator/trace-server/rcp/?d) or let `yarn` download and run it:
+In order to open traces, you need a trace server running on the same machine as the trace extension. There are 3 options to run the trace server.
+
+#### Using yarn (Linux only)
+
+If you are working on Linux, you can let `yarn` download and run it:
 
 ```bash
 yarn download:server
 yarn start:server
 ```
 
+#### Manual download (Windows, Linux and MacOS)
+
+Alternatively, you can download the [Eclipse Trace Compass server](https://download.eclipse.org/tracecompass.incubator/trace-server/rcp/?d). Select the version of the trace server that best matches your working environment. There are different versions of the trace server for Linux, Windows and MacOS.
+
+After you have successfully downloaded the Trace Server, extract the compressed files, rename the extracted folder to `trace-compass-server`, then copy it into the root of the repo.
```

Comment:
After extracting the server, the user can update the preference I mentioned earlier to allow the application to start the server automatically. Then you don't need to rename it.